.github/CONTRIBUTING.md

```diff
@@ -154,18 +154,18 @@ Do not compare boolean values to TRUE or FALSE. For TRUE you should just check i
 // Bad
 var/thing = pick(list(TRUE, FALSE))
```

Comment:
Not sure if it's in scope to change this, but we don't need to put things in lists for `pick()` to work

Reply:
Yeah, I think its an example of bad code

Reply:
Yeah, might want to do that in a separate PR, could probably be worth a once over once this goes in

Reply:
I'll open a PR to change it tonight then